pettingzoo/classic/backgammon/backgammon_env.py

```diff
@@ -20,9 +20,10 @@ def env(**kwargs):
 
 class raw_env(AECEnv):
     metadata = {
-        'render.modes': ['human'],
+        "render.modes": ["human"],
         "name": "backgammon_v3",
         "is_parallelizable": False,
+        "video.frames_per_second": 2,
```

Comment:
Why is it 2 frames per second? I don't recall any FPS style logic anywhere in these classic environments, unless I am forgetting something.

Reply:
Indeed, there is actually no need for an FPS in these environments (they are turn based afterall). But the FPS metadata is here just for the end user to use in case they intend to view things as a clip with the .render() function. I tested a few different FPSes and have also looked at the gifs in the Pettingzoo website, and eyeballed that 2 FPS seems to look decent.